### ISSUE TITLE
[1.20.6] ATs still use SRG mappings

### ISSUE THREAD

**Issue by PaintNinja** [MinecraftForge/MinecraftForge#9954]
**Minecraft Version:** 1.20.6

**Forge Version:** 50.0.2

**Logs:** N/A

**Steps to Reproduce:**
 1. Clone Forge
 2. Check the accesstransformers.cfg file
 3. Observe that they're not using official/MojMap mappings for the field and method names

**Description of issue:**
The move to MojMaps everywhere is great, although ATs still appear to be mapped with SRG. If this could be changed to use MojMap that'll be great

**Comment by LexManos** [comment]
Correct, the problem is that it requires tooling changes. Namely that ATs are applied pre-decompile. And pre-decompile is in SRG names. We would either need to build a system into Forge's buildscripts to convert the config to SRG before decompile. Or teach AT about Mapped names. 

If we were to move Forge's ATs to Official we would need to bake them to SRGs for userdev like we do the patch files.

So ya, I would like to do this but toolchain needs work.
